src/protocolgame.cpp

```diff
@@ -3148,18 +3150,18 @@ void ProtocolGame::sendOutfitWindow()
 	writeToOutputBuffer(msg);
 }
 
-void ProtocolGame::sendPodiumWindow(const Item* item)
+void ProtocolGame::sendPodiumWindow(std::shared_ptr<const Item> item)
 {
 	if (!item) {
 		return;
 	}
 
-	const Podium* podium = item->getPodium();
+	auto podium = item->getPodium();
```

Comment:
Could this be `const auto`
```suggestion
	const auto podium = item->getPodium();
```